xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-webjar/src/main/less/lightbox.less

```diff
@@ -0,0 +1,82 @@
+/*! #template('colorThemeInit.vm') */
```

Comment:
No, don't mix LESS with Velocity. Just keep the LESS code as is so that it gets compiled at runtime.